include/cuco/util/key_generator.hpp

```diff
@@ -34,21 +34,36 @@
 #include <time.h>
 #include <type_traits>
 
-namespace cuco::benchmark {
+namespace cuco::util {
 
 namespace dist_type {
```

Comment:
Actually, is `distribution` better than `dist_type`?

Reply:
Yep, that's a remnant from the prototype. I'll change it to `distribution`

Reply:
addressed in [5bfa854](https://github.com/NVIDIA/cuCollections/pull/275/commits/5bfa854e95eb87c07e1dbde0f77c249830678685)